sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/SessionRecords/CassandraResourcesOperationsTests/CassandraCRUDTestsAsync()Async.json

```diff
@@ -4,7 +4,6 @@
       "RequestUri": "https://management.azure.com/subscriptions/db1ab6f0-4769-4b27-930e-01e2ef9c123c/resourcegroups/Default-CosmosDB-8918?api-version=2017-05-10",
       "RequestMethod": "PUT",
       "RequestHeaders": {
-        "Accept": "application/json",
```

Comment:
Did you manually do this or merged master and regenerated?

Reply:
Regenerated